examples/apps/attributable-map/webpack.config.cjs

```diff
@@ -32,7 +32,7 @@ module.exports = (env) => {
 			},
 			output: {
 				filename: "[name].bundle.js",
-				path: path.resolve(__dirname, "dist"),
+				path: path.resolve(__dirname, "lib"),
```

Comment:
We should not mix tsc output and webpack output. Prefer a rename to have a webpack directory, but okay to leave as dist for a later general webpack config cleanup.
You can test this out by webpacking (just build or just serve(start)) without doing any tsc.